modal_proto/api.proto

```diff
@@ -1248,11 +1248,17 @@ message FunctionInput {
 }
 
 message FunctionMapRequest {
+  enum InvokeType {
+    INVOKE_TYPE_UNSPECIFIED = 0;
+    INVOKE_TYPE_SYNC = 2;
+    INVOKE_TYPE_ASYNC = 1;
```

Comment:
Would something else be more clear like `INVOKE_TYPE_BLOCKING` and `INVOKE_TYPE_NON_BLOCKING`?

Reply:
We can change it later. SYNC and ASYNC is how Lamdba differentiates its execution patterns so we can stick with that for now.